api/app/package.json

```diff
@@ -27,6 +27,7 @@
     "dayjs": "^1.11.6",
     "dotenv": "^16.0.3",
     "express": "^4.18.1",
+    "express-http-proxy": "^1.6.3",
```

Comment:
I tried to use https://www.npmjs.com/package/http-proxy and https://www.npmjs.com/package/http-proxy-middleware with no success, so went with the next one in popularity that fit our use case.